### ISSUE TITLE
humpty is failing with the new version of distlib (0.2.5)

### ISSUE THREAD

**Issue by kalessin** [dairiki/humpty#1]
It is fixed when forcing distlib 0.2.4. So one fast solution is to pin required version to 0.2.4, and then re publish on pypi, as the package as installed by pip  will always fail, because always retrieve last version of distlib.

**Comment by kalessin** [comment]
btw this is the traceback

```
Traceback (most recent call last):
  File "/usr/local/bin/humpty", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/humpty.py", line 620, in main
    EggWriter(wheel).build_egg(dist_dir)
  File "/usr/local/lib/python2.7/site-packages/humpty.py", line 521, in build_egg
    for filename, content in egg_info:
  File "/usr/local/lib/python2.7/site-packages/humpty.py", line 111, in files
    sections = getattr(self, name)
  File "/usr/local/lib/python2.7/site-packages/humpty.py", line 275, in requires
    reqs = list(get_reqs())
  File "/usr/local/lib/python2.7/site-packages/humpty.py", line 268, in get_reqs
    req_, sep, marker = req.rpartition(';')
AttributeError: 'dict' object has no attribute 'rpartition'
```

**Comment by dairiki** [comment]
I believe that this is fixed in c1b290e (which is included in the just-released humpty 0.2.)
